SimTKcommon/include/SimTKcommon/internal/ClonePtr.h

```diff
@@ -202,7 +207,11 @@ template <class T> class ClonePtr {
         other.reset(p);
         reset(otherp);
     }
-     
+
+    DEPRECATED_14("use get() instead")
+    const T* getPtr()  const  { return get(); }
+    DEPRECATED_14("use upd() instead")
+    T* updPtr() { return upd(); }
```

Comment:
> What about using the macro to also insert a doxygen comment for the method?

What would you think about instead hiding deprecated things from doxygen? Is there a good reason to keep documenting them if we don't want people to use them?

I did consider a permanent `SimTK_DEPRECATED` also. Another reason for that might be if we want to alter the reported message. Currently I am inserting "DEPRECATED:" before the supplied message because Microsoft just outputs `Warning C4996` which doesn't exactly scream "deprecated". But ... I'm hoping for nicer behavior when they implement the standard. And I'm not a big fan of macros. It made me very happy to get rid of the `OVERRIDE_11` macro.